src/content/recipes/vuetify.md

```diff
@@ -0,0 +1,196 @@
+<div class="aside aside__no-top">
+
+This recipe assumes that you have a Vue 3 app using Vuetify v3 and have just set up Storybook 7.0 using the [getting started guide](https://storybook.js.org/docs/7.0/vue/get-started/install). Don’t have this? Follow Vuetify’s [installation instructions](https://next.vuetifyjs.com/en/getting-started/installation/#installation) then run:
+
+```shell
+# Add Storybook:
+npx sb@next init --prerelease
+```
+
+</div>
+
+<RecipeHeader>
+
+How to setup Vuetify and Storybook
+
+</RecipeHeader>
+
+Vuetify is a popular UI framework for Vue.js that provides a variety of pre-designed components, while Storybook is a tool for creating and testing UI components in isolation.
+This post will show you how to integrate these two tools to create a powerful and flexible development environment for building user interfaces with Vuetify.
+
+This post will explain how to:
+
+- 🔌 Setup Vuetify with Storybook
+- 🧱 Use Vuetify in your components
+- 🎨 Switch Vuetify themes in a click
+
+If you’d like to see the example code of this recipe, check out the [example repository](https://github.com/Integrayshaun/vue3-vuetify-storybook-recipe-example) on GitHub. Let's get started!
+
+![Completed Vuetify example with theme switcher](https://user-images.githubusercontent.com/18172605/207120625-bedb53ec-eac4-4690-a06a-5d0579cb9809.gif)
+
+## Register Vuetify in Storybook
+
+To get started, you'll need to add Vuetify’s fontloader and plugin to your Storybook configuration.
+To do this, add the following to your `.storybook/preview.js` file:
+
+```js
+// .storybook/preview.js
+
+import { setup } from '@storybook/vue3';
+import { registerPlugins } from '../src/plugins';
+
+setup((app) => {
+  // Registers your app's plugins into Storybook
+  registerPlugins(app);
+});
+```
+
+Here `registerPlugins` loads Vuetify’s fonts and registers all of its components with Storybook’s Vue app.
+
+## Using Vuetify Components
+
+Let’s update some of our example components to use Vuetify instead. Open up the Button component in `./src/stories/button.vue`.
+
+![Unchanged example button component from Storybook init](https://user-images.githubusercontent.com/18172605/207120859-0383d01a-6448-4327-94c9-dcb1ec86f868.png)
+
+Currently, it’s not looking very Vuetiful so let’s make some changes. Replace the contents of `./src/stories/Button.vue` with the following code:
+
+```vue
```

Comment:
☹️ 🐼  for lack of syntax highlighting with Vue SFCs.

Reply:
Yeah :( it's a bummer. I'll look into adding support though